File: benchmark/corpus/03_div_zero/e_range_width.py
# Divides by hi - lo; input gives the same number twice.
import sys


def density(total, lo, hi):
    return total // (hi - lo)  # DANGER


parts = sys.stdin.readline().strip().split(",")
if len(parts) != 2:
    print("need two numbers")
else:
    lo, hi = int(parts[0]), int(parts[1])
    print("density %d" % density(1000, lo, hi))

File: benchmark/corpus/03_div_zero/e_range_width.js
// Divides by hi - lo; input gives the same number twice.
function density(total, lo, hi) {
  return Math.floor(total / (hi - lo)); // DANGER
}

const parts = (await new Response(Deno.stdin.readable).text()).trim().split(",");
if (parts.length !== 2) console.log("need two numbers");
else console.log("density " + density(1000, Number(parts[0]), Number(parts[1])));
